python/tk_multi_publish2/dialog.py

```diff
@@ -268,7 +268,7 @@ def _on_item_comment_change(self):
 
                 # this is the summary item - so update all items!
                 for top_level_item in self._plugin_manager.top_level_items:
-                    top_level_item.description = self._summary_comment
+                    top_level_item._propagate_description_to_children(self._summary_comment)
```

Comment:
The method is oddly named. It's called `_propagate_description_to_children`, but it operates both on the item AND its children. I think the method should only update children and add back in the original line 271. This would make the intent clearer.